build.gradle

```diff
@@ -726,6 +726,7 @@ ext.createDiffFile = { ->
         ':!services/venice-standalone/*', // exclude the entire standalone project
         ':!clients/venice-client/src/main/java/com/linkedin/venice/fastclient/factory/ClientFactory.java',
         ':!clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java',
+        ':!clients/da-vinci-client/src/main/java/com/linkedin/davinci/storage/StorageService.java',
```

Comment:
Why not write some unit test against this class?

Reply:
sure

Reply:
actually its bit complex throw the rocksDBException as the rocksdb engine is opened several level under storage service and hard to mock it.

Reply:
Can we just mock the behavior of `openStore` so we don't need to mock rocksDB's behavior?

Reply:
We would need to add a surgery method for `persistenceTypeToStorageEngineFactoryMap` to put in a mocked version of `RocksDBStorageEngineFactory` and store engine map to return mock of `RocksDBStorageEngine` which is doable, but will add too many test only design in the main class.